#!/usr/bin/env python
PACKAGE = "maxon_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#gen.add("speed", int_t, 0, "Speed of motor",    0, 0,  300)
gen.add("cw_rotation",   bool_t,   0, "Clockwise rotation of motor",  True)

gen.add("KP", double_t, 0, "Proportional gain",    0.018, 0,  5)
gen.add("KI", double_t, 0, "Integral gain",    0.00003, 0,   5)
gen.add("KD", double_t, 0, "Derivative gain",    0, 0,   20)


exit(gen.generate(PACKAGE, "maxon_control", "Parameters"))
